.github/ISSUE_TEMPLATE/new_exchange.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Add an exchange
+about: Suggest a new exchange for us to list
+title: ""
+labels: "Type: Feature"
+assignees: ""
+---
+
+Please fill out this short form if you'd like an exchange added to ethereum.org. If it's a decentralized exchange (DEX) you'd like to list, please take a look at our [policy for listing dapps](https://ethereum.org/en/contributing/adding-products/).
+
+**What geographical restrictions apply to this exchange?** 
+
+<!-- If you don't know, please get in touch with the exchange. They'll likey have a list of restricted countries. -->
+
+**Which currencies are accepted by this exchange?**
+
+<!-- If you don't know, please get in touch with the exchange. They'll likey have a list of accepted currencies -->
+
+**Do you have any proof that they're an official company**
```

Comment:
I'm not sure about the wording of "official company" - I think "legitimate company" might be more what we are getting at.

Reply:
Agreed, "legitimate" or "credible".